clock_exercise/clockmanager.h

```diff
@@ -15,6 +15,7 @@ class ClockManager : public QObject
     Q_PROPERTY(float timerCurrentSeconds MEMBER m_timerCurrentSeconds NOTIFY timerCurrentSecondsChanged WRITE setTimerCurrentSeconds)
     Q_PROPERTY(int secondsInOneMinute MEMBER SECONDS_IN_ONE_MINUTE CONSTANT)
     Q_PROPERTY(int minutesInOneHour MEMBER MINUTES_IN_ONE_HOUR CONSTANT)
+    Q_PROPERTY(int hoursInADay MEMBER HOURS_IN_A_DAY CONSTANT)
```

Comment:
A mio parere non ha molto valore mettere questa property, non mi aspetto che le ore e i minuti in un giorno cambino nel tempo di vita dell'applicazione...
Potremmo fare un discorso analogo per i secondi, perché per la gran parte delle applicazioni reali possiamo ignorare i leap seconds.

Reply:
MinutesInOneHour e hoursInADay sono state eliminate, secondsInOneMinute lo uso per convertire timerCurrentSeconds in minuti, la cosa rischia di diventare poco comprensibile se tolgo anche questa costante.